packages/buttons/src/styled/StyledButton.ts

```diff
@@ -209,9 +209,17 @@ const sizeStyles = (props: IStyledButtonProps & ThemeProps<DefaultTheme>) => {
   } else {
     const height = getHeight(props);
     const lineHeight = math(`${height} - (${props.theme.borderWidths.sm} * 2)`);
-    const padding = props.theme.space.base * 7;
+    let padding;
     let fontSize;
 
+    if (props.size === 'small') {
+      padding = `${props.theme.space.base * 3}px`;
+    } else if (props.size === 'large') {
+      padding = `${props.theme.space.base * 5}px`;
+    } else {
+      padding = `${props.theme.space.base * 4}px`;
+    }
```

Comment:
Now that this is in-context, we see a further opportunity for optimization. There's no need to travel through the conditionals twice when both `padding` and `fontSize` can be resolved like this:

```js
if (props.size === SIZE.SMALL) {
  fontSize = props.theme.fontSizes.sm;
  padding = `${props.theme.space.base * 3}px`;
} else {
  fontSize = props.theme.fontSizes.md;

  if (props.size === SIZE.LARGE) {
    padding = `${props.theme.space.base * 5}px`;
  } else {
    padding = `${props.theme.space.base * 4}px`;
  }
}
```

Also notice I used the `SIZE` constants vs strings.

Reply:
The strings part was one thing I was so curious about! Is that a preference thing or is one a better option?

Will push up this update right meow. Thanks for working through this with me. Learning a lot.

Reply:
If they are easily available, I think using constants is always preferable. The good news is TS has us covered when they aren't. I think the original code here was just an oversight re: constants vs strings.